src/ui/handler/scroll_zoom.js

```diff
@@ -70,6 +78,22 @@ class ScrollZoomHandler {
         ], this);
     }
 
+    /**
+     * Override default zoomRate value
```

Comment:
How about something like `Set the zoom rate of a trackpad`?

We should also add the default here so that people reading the docs can know what sort of number to enter. You can include a default like this `@param {number} [zoomRate=1/100]`